bibliopixel/drivers/APA102.py

```diff
@@ -1,23 +1,19 @@
-from spi_driver_base import DriverSPIBase, ChannelOrder
+from . spi_driver_base import DriverSPIBase, ChannelOrder
 
 
 class DriverAPA102(DriverSPIBase):
     """Main driver for APA102 based LED strips on devices like the Raspberry Pi and BeagleBone"""
 
-    def __init__(self, num, c_order=ChannelOrder.RGB, use_py_spi=True, dev="/dev/spidev0.0", SPISpeed=2):
-        super(DriverAPA102, self).__init__(num, c_order=c_order,
-                                           use_py_spi=use_py_spi, dev=dev, SPISpeed=SPISpeed)
+    def __init__(self, num, **kwds):
```

Comment:
I think I get what you are doing, but I really don't like this. I realize that the __init__ methods are often REALLY verbose but it also improves readability and auto-filling input parameters in editors. With it like this, you'd have to go look in the docs... I like self documenting code :) At the very least, we need a docstring to replace the removed params.